packages/@aws-cdk/aws-batch/test/integ.batch.ts

```diff
@@ -0,0 +1,55 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import ecr = require('@aws-cdk/aws-ecr');
+import { ContainerImage, EcrImage } from '@aws-cdk/aws-ecs';
```

Comment:
Can you `import * as ecs from '@aws-cdk/aws-ecs'` instead? (Yes, this PR has been in review for so long that we've actually changed our import style in the meantime 😃

Reply:
Hahaha no worries. Its been a **LONG** time but we will make it through :P better practice tbh